mscore/palette.h

```diff
@@ -134,13 +155,13 @@ class Palette : public QWidget {
       virtual void leaveEvent(QEvent*) override;
       virtual bool event(QEvent*) override;
       virtual void resizeEvent(QResizeEvent*) override;
+      void applyPaletteElement(PaletteCell* cell);
 
       virtual void dragEnterEvent(QDragEnterEvent*) override;
       virtual void dragMoveEvent(QDragMoveEvent*) override;
       virtual void dropEvent(QDropEvent*) override;
       virtual void contextMenuEvent(QContextMenuEvent*) override;
 
-      int idx(const QPoint&) const;
```

Comment:
It's needed in PaletteBox::mousePressEvent so that a palette element can be selected using the mouse.